administration/config/env.ts

```diff
@@ -56,7 +56,7 @@ const getClientEnvironment = (
   const appDirectory = fs.realpathSync(process.cwd())
   process.env.NODE_PATH = (process.env.NODE_PATH || '')
     .split(path.delimiter)
-    .filter(folder => folder && !path.isAbsolute(folder))
+    .filter(folder => !!folder && !path.isAbsolute(folder))
```

Comment:
I think you are testing for zero length strings here? If that is the case, I would use `folder.length > 0`. It is slightly longer but way more explicit.